tests/e2e/tests/test_profile_irsa.py

```diff
@@ -314,3 +327,35 @@ def test_notebook_irsa(self, notebook_server, region):
         assert bucket_name in buckets
 
         s3_client.delete_bucket(Bucket=bucket_name)
+    
+    def test_pipeline_component_irsa(self, kfp_client, client_namespace):
+
+        def s3_op():
+            import boto3
+            import random
+            import string
+
+            suffix = "".join(
+                random.choice(string.ascii_lowercase + string.digits) for _ in range(10)
+            )
+            s3_bucket_name = "test-pipeline-profile-irsa-" + suffix
+
+            s3 = boto3.client("s3", region_name="us-west-2")
```

Comment:
why is this hard-coded to us-west-2?

Reply:
s3 buckets can be accessed in any region so hosted region doesn't matter for testing